src/main/java/seedu/apollo/exception/task/DateOverException.java

```diff
@@ -0,0 +1,40 @@
+package seedu.apollo.exception.task;
+
+import java.time.LocalDateTime;
+
+import static seedu.apollo.task.Task.printPattern;
+
+public class DateOverException extends Throwable {
+
+    String type;
+    String description;
+    LocalDateTime by;
+    LocalDateTime from;
+    LocalDateTime to;
```

Comment:
Perhaps you could assign the accessibility to these variables

Reply:
I have mentioned this already